### PR TITLE
Rebuild proto artifacts from existing sources and update README.md instructions.

### PR DESCRIPTION
**Pull request by mikelehen**
This is the result of me updating README.md and following the steps in it.  Some of the diffs were for a proto change Greg made a while ago.  I'm not really sure why Firestore.pbrpc.h and Firestore.pbrpc.m changed.  Everything seems to still build, but if you suspect I did something wrong, let me know. I've never done this before.

I'm doing this in preparation for /actually/ updating the protos.

### CODE REVIEW

**Comment by wilhuff** [comment]
The spurious diff you've encountered comes from the fact that the generation strategy for imports [changed](https://github.com/grpc/grpc/commit/9b7484867663018e2b362cf01567dd9a77341f00). It was released with gRPC 1.8.0, which went live [12/12/2017](https://github.com/grpc/grpc/releases/tag/v1.8.0).

I last generated these files before our initial release which was merged into this repo in bde743ed25166a0b320ae157bfb1d68064f531c9 on 10/3/2017. I attempted to regenerate this in January in bc74670afec651c3f912cb6b7e54f5b68bd507f5, but rolled it back after the change didn't work as expected. That change included these diffs.

It's likely that when @rsgowman was working in here on adding nanopb support he just reverted these diffs as being unrelated?

Basically: I don't see anything wrong with this.

**Comment by mikelehen** [comment]
Thanks!  If you don't see anything wrong, could you approve it then? :-)  Or is there something extra I should do first?  [I'll have another PR that actually updates the protos, but it'll be good to get these diffs checked in separately I think...]
